README.md

```diff
@@ -12,3 +12,9 @@ The `App` renders an `Article` and passes the data from the `article` object.
 
 ***Extra task (optional):***
 Add props validation using `PropTypes` library and `defaultProps`
+ ## REQUIREMENTS:
+ 1. Page "Article" should contain these attributes:
+ - `data-cy="title"` attribute inside `h1` tag;
+ - `data-cy="date"` attribute inside a `span` tag;
+ - `data-cy="text"` attributeinside `p` tag.
```

Comment:
Why do we need it? In this case better to search by attributes. We need to check that the student uses the correct tag. And do not need to complicate student's job without needs